apps/wing/tsconfig.json

```
@@ -21,8 +21,8 @@
     "emitDecoratorMetadata": true,
     "declaration": true,
     "baseUrl": ".",
-    "esModuleInterop": true,
+    "esModuleInterop": true
   },
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "src/test/*"],
```

This shouldn't be needed afaik

```suggestion
  "include": ["src/**/*"],
```